content/en/blog/_posts/2021-08-04-kubernetes-release-1.22.md

```diff
@@ -0,0 +1,159 @@
+
+For the full list, check out the [Deprecated API Migration Guide](https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-22) as well as the blog post [Kubernetes API and Feature Removals In 1.22: Here’s What You Need To Know](https://blog.k8s.io/2021/07/14/upcoming-changes-in-kubernetes-1-22/). 
+
+###  API changes and improvements for ephemeral containers
+
+The API used to create [Ephemeral Containers](https://kubernetes.io/docs/concepts/workloads/pods/ephemeral-containers/) changes in 1.22. The Ephemeral Containers feature is alpha and disabled by default, and the new API does not work with clients that attempt to use the old API.
+
+For stable features, the kubectl tool follows the Kubernetes [version skew policy](https://kubernetes.io/releases/version-skew-policy/); however, kubectl v1.21 and older do not support the new API for ephemeral containers. Users who create ephemeral containers using `kubectl debug` should note that kubectl version 1.22 will attempt to fall back to the old API; older versions of kubectl will not work with cluster versions of 1.22 or later. Please update kubectl to 1.22 if you wish to use `kubectl debug` with a mix of cluster versions.
+
+## Other Updates
+
+### Graduated to Stable
+
+* [Bound Service Account Token Volumes](https://github.com/kubernetes/enhancements/issues/542)
+* [CSI Service Account Token GA](https://github.com/kubernetes/enhancements/issues/2047)
+* [Support CSI Plugins in Windows](https://github.com/kubernetes/enhancements/issues/1122)
+* [Warning mechanism for deprecated API use](https://github.com/kubernetes/enhancements/issues/1693)
+* [Pod Disruption Budget Eviction](https://github.com/kubernetes/enhancements/issues/85)
+* [CronJobs](https://github.com/kubernetes/enhancements/issues/19)
+
+### Notable Feature Updates
+
+* [Memory Manager](https://github.com/kubernetes/enhancements/issues/1769)
+* [Generic data populators DataSource](https://github.com/kubernetes/enhancements/issues/1495)
+* [kubeadm configuration](https://github.com/kubernetes/enhancements/issues/970)
+* [API Server Tracing](https://github.com/kubernetes/enhancements/issues/647)
+* [Rootless mode containers](https://github.com/kubernetes/enhancements/issues/2033)
+* [PodSecurity admission](https://github.com/kubernetes/enhancements/issues/2579) (PodSecurityPolicy replacement)
+
+# Release notes
+
+You can check out the full details of the 1.22 release in the [release notes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.22.md).
+
+# Availability of release
+
+Kubernetes 1.22 is available for [download on GitHub](https://github.com/kubernetes/kubernetes/releases/tag/v1.22.0).
```

Comment:
As a new feature, we can link to https://kubernetes.io/releases/download/ as well, if we want to.